src/pages/Checklist/index.tsx

```diff
@@ -246,80 +280,127 @@ export const Checklist = () => {
           </PortTable>
         </section>
         <section>
-          <Heading level={3}>Configure time sync</Heading>
-          <Text className="mt20">For Ubuntu 20.04</Text>
+          <Heading level={3}>
+            <FormattedMessage defaultMessage="Configure time sync" />
+          </Heading>
+          <Text className="mt20">
+            <FormattedMessage
+              defaultMessage="For {operatingSystem}"
+              values={{
+                operatingSystem: 'Ubuntu 20.04',
+              }}
+              description="Indicates which operating system the instructions are for"
+            />
+          </Text>
           <ul className="sub-checklist-item">
             <li className="py5">
-              <Text>Run the following command</Text>
+              <Text>
+                <FormattedMessage defaultMessage="Run the following command" />
+              </Text>
               <CodeSnippet>
                 <code>timedatectl</code>
               </CodeSnippet>
             </li>
             <li className="py5">
               <Text>
-                Check if <Code>NTP Service</Code> is <Code>active</Code>.
+                <FormattedMessage
+                  defaultMessage="Check if {code1} is {code2}."
+                  values={{
+                    code1: <Code>NTP Service</Code>,
+                    code2: <Code>active</Code>,
+                  }}
+                  description="{code} values are terminal outputs to look for, should not translate"
+                />
               </Text>
             </li>
             <li className="py5">
               <Text>
-                Check if <Code>Local time</Code>, <Code>Time zone</Code>, and{' '}
-                <Code>Universal time</Code> are all correct.
+                <FormattedMessage
+                  defaultMessage="Check if {code1}, {code2}, and {code3} are all correct."
+                  values={{
+                    code1: <Code>Local time</Code>,
+                    code2: <Code>Time zone</Code>,
+                    code3: <Code>Universal time</Code>,
+                  }}
+                  description="{code} values are terminal outputs to look for to confirm system time"
+                />
```

Comment:
@samajammin honestly not sure if these should be translated or not... I would think a user's system settings would produce these in their native language, but not sure.

Reply:
Good question. @CarlBeek do you know anyone who might know this? Does `timedatectl ` support multiple languages? A quick Google didn't yield any aswers.

Reply:
If in doubt I suggest we keep it in English.

Reply:
Okay... this is set up to not be translated at the moment, I will leave it pending any new information.

Reply:
Merging this for now. @CarlBeek please let us know if you're aware of any `timedatectl` language support.